src/main/java/uk/gov/hmcts/ccd/domain/service/processor/DisplayContextParameter.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.ccd.domain.service.processor;
```

Comment:
DisplayContextParameter it's a core domain concept that should be outside the processor package